docs/index.rst

```diff
@@ -6,8 +6,8 @@ Toga is a Python native, OS native, cross platform GUI toolkit. Toga consists of
 library of base components with a shared interface to simplify platform-agnostic GUI
 development.
 
-Toga is available on macOS, Windows, Linux (GTK), Android, iOS, and for
-single-page web apps.
+Toga is available on macOS, Windows, Linux (GTK), Android, iOS, for
+single-page web apps, and console apps.
```

Comment:
What? Of course I believe in Oxford commas... when have I not?

Reply:
This may be a temporal memory bias where I saw a missing oxford comma elsewhere....but I've noticed generally we have distinct preferences for commas...especially around using them with conjunctions for clauses. Nonetheless, this was only in jest...the only real rule I have for english anymore is whether a statement can be unambiguously interpreted and understood.